### PR TITLE
Bloodlust - new texture pack (updated)

### PR DESCRIPTION
**Pull request by Morgothek**
Bloody

### CODE REVIEW

File: texturepacks/Bloodlust/Bloodlust.rar
Comment (itzTheMeow) [inline texturepacks/Bloodlust/Bloodlust.rar:1]: delete this?

File: texturepacks/Bloodlust/_original_c.png
Comment (itzTheMeow) [inline texturepacks/Bloodlust/_original_c.png:1]: seems random

**Review by itzTheMeow — CHANGES_REQUESTED** [review]
delete the rar and review the _original png file

File: texturepacks/Bloodlust/Bloodlust.rar
Comment (Morgothek) [reply texturepacks/Bloodlust/Bloodlust.rar:1]: done

**Review by itzTheMeow — APPROVED** [review]

**Pull request merged**